index.html

```diff
@@ -144,5 +144,25 @@ <h3 class="subtitle">Skills</h3>
             </ul>
         </section>
     </section>
+
+    <section class="contactContainer">
+        <h2 class="title blue">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form class="contactForm" action="https://formspree.io/f/mzbwkvzy" method="POST">
+            <input class="singleLineInput" name="name" type="text" placeholder="Full Name" maxlength="30" required>
+            <input class="singleLineInput" name="email" type="email" placeholder="Email address" required>
+            <textarea class="commentArea" name="comments" placeholder="Enter text here" maxlength="500" required></textarea>
+            <button class="button">
+                Get in touch
+            </button>
+        </form>
+        <ul class="mediaContainer">
+            <li><a href="https://twitter.com/Victor_GonMa" target="_blank" ><img class="mediaIcon" src="images\twitter.png" alt="Twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/victor-manuel-gonzalez-massimi-a77265124/" target="_blank" ><img class="mediaIcon" src="images\Linkedin icon.png" alt="Twitter"></a></li>
+            <li><a href="https://github.com/Vgonma" target="_blank" ><img class="mediaIcon" src="images\medium.png" alt="Twitter"></a></li>
```

Comment:
- [ ] Great job on implementing the contact form, but make sure your design sticks to your chosen Figma template. Kindly fix :wrench:  the background image in this section, your design to look like your chosen template.

|Figma design| Your design |
|------------------|-----------------|
|![Screenshot from 2022-11-02 01-20-12](https://user-images.githubusercontent.com/65129228/199353304-7b2d049e-1f96-44a5-a56d-ddd4ba977240.png)|![Screenshot from 2022-11-02 01-23-14](https://user-images.githubusercontent.com/65129228/199353390-1b340db8-250a-493b-afc7-9ff75e1170e2.png)|